wandb/apis/__init__.py

```diff
@@ -9,3 +9,7 @@
 
 from .internal import Api as InternalApi
 from .public import Api as PublicApi
+
+# Is there a better way to get public modules into wandb_artifacts?
+# I feel like I should not need to to this.
+from .public import Artifact as PublicArtifact
```

Comment:
Also, `PublicApi` is already exposed. You can construct it and then call a artifact() on it. `api = PublicApi(); api.artifact()`.

Reply:
The root of the motivation was not to introduce new endpoints int he user-facing API. As you recall, I had a helper function in public.py's `API` class. The reason it was there was twofold:
1) the `from_id` factory needs a client, so it was convenient to pass `self` from the API layer
2) I wanted to avoid circular imports.

I think a cleaner approach is to decouple API from the Artifact construction, but to do so requires this awkward import. 

What are your thoughts?

Reply:
cc @shawnlewis from his comment below

Reply:
Maybe in sdk/wandb_artifacts.py we just do:
`from wandb.apis.public import Artifact as PublicArtifact`, and get rid of the change to wandb/apis/__init__.py ?

Reply:
This works - incoming on next commit.